hazelcast-client-legacy/src/test/java/com/hazelcast/client/multimap/ClientMultiMapLockTest.java

```diff
@@ -324,8 +328,13 @@ public void testLockTTLExpired_whenLockedBySelf() throws Exception {
 
         mm.lock(key);
         mm.lock(key, 1, TimeUnit.SECONDS);
-        sleepSeconds(2);
-        assertFalse(mm.isLocked(key));
+
+        assertTrueEventually(new AssertTask() {
```

Comment:
I would introduce a method 'assertIsUnlockedEventually to prevent copy/paste. Apart from that it looks like good test improvements.